app.js

```diff
@@ -2,31 +2,29 @@
 
   function Product(name) {
   this.name = name;
```

Comment:
I like that you've made two out of the three properties you need work with just one property, but best practice is to have a name and a separate image path. Don't forget to add a number of views property.